src/test/MsQuicTests.h

```diff
@@ -811,6 +814,10 @@ typedef struct {
     QUIC_CTL_CODE(27, METHOD_BUFFERED, FILE_WRITE_DATA)
     // int - Family
 
+#define IOCTL_QUIC_RUN_VALIDATE_NET_STATS_CONN_EVENT \
+    QUIC_CTL_CODE(28, METHOD_BUFFERED, FILE_WRITE_DATA)
```

Comment:
Add the new IOCTL at the end of the file and use the next number available (not 28, as it's already used). Then update the MAX define at the end. Then you will need to update `QUIC_IOCTL_BUFFER_SIZES` in `control.cpp`.